epgarcia/liferay-portal
# LPS-100500

On top of https://github.com/epgarcia/liferay-portal/tree/pr-943
* Validation split 0-1
* Check editable in run experiment service
* Add Integration test
